src/App.css

```diff
@@ -1,41 +1,99 @@
-#root {
-  max-width: 1280px;
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  color: darkslategray;
+}
+
+body {
+  background: steelblue;
+  max-width: auto;
+  padding: 1rem;
+  margin: 0 auto;
+}
+
+.header {
+  color: darkslategray;
   margin: 0 auto;
-  padding: 2rem;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+#root {
+  background-color: aliceblue;
+  max-width: fit-content;
+  margin: 4rem auto;
+  padding: 2rem;
+  border-radius: 0.5em;
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+.btn {
```

Comment:
To reduce class proliferation, given this class is only used on `button` elements, just use the element selector directly. You can then remove this class from the buttons in the JSX